airflow/example_dags/example_http_operator.py

```diff
@@ -89,10 +94,7 @@
     request_params={},
     response_check=lambda response: True if "Google" in response.content else False,
     poke_interval=5,
-    dag=dag)
+    dag=dag,
+)
 
-t1.set_upstream(sensor)
-t2.set_upstream(t1)
-t3.set_upstream(t2)
-t4.set_upstream(t3)
-t5.set_upstream(t4)
+sensor >> t1 >> t2 >> t3 >> t4 >> t5
```

Comment:
This is a clear example why I prefer `sensor >> t1 >> t2 >> t3 >> t4 >> t5` over:
```python
t1.set_upstream(sensor)
t2.set_upstream(t1)
t3.set_upstream(t2)
t4.set_upstream(t3)
t5.set_upstream(t4)
```